src/config/index.ts

```diff
@@ -164,9 +164,16 @@ export const SENTRY_REPOS: Set<string> = new Set([
 ]);
 
 /**
- * Personal Access Token for the Sentry bot used to do things that aren't possible with the App account, e.g. querying org membership
+ * As far as we can tell, it's not possible to check private org membership
+ * from an app installation. Therefore, we use a Personal Access Token for a
+ * bot account that is itself an org member.
+ *
+ * If you set FORCE... we will *always* use user auth instead of app
+ * installation auth, to make dev life easier.
  */
 export const GH_USER_TOKEN = process.env.GH_USER_TOKEN || '';
+export const FORCE_USER_TOKEN_GITHUB_CLIENT =
+  process.env.FORCE_USER_TOKEN_GITHUB_CLIENT == 'true';
```

Comment:
```suggestion
  process.env.FORCE_USER_TOKEN_GITHUB_CLIENT == 'true' || process.env.FORCE_USER_TOKEN_GITHUB_CLIENT == '1';
```

Reply:
Done in 836c0050c20a4d9d9a3d800799df4697aa068ef8.